tests/bearlib/aspects/InstanceTest.py

```diff
@@ -1,4 +1,4 @@
-from itertools import chain
+from itertools import chain  # Ignore PyUnusedCodeBear
```

Comment:
I don't think we need this import either.

@arjunsinghy96 actually, it'd be great if you could check if any of these commented import lines are even needed. If they are not, you could remove them (in a different commit that preceeds this commit).